pkg/util/bandwidth/linux.go

```diff
@@ -224,6 +224,12 @@ func (t *tcShaper) interfaceExists() (bool, string, error) {
 	if len(value) == 0 {
 		return false, "", nil
 	}
+	// Newer versions of tc and/or the kernel return the following instead of nothing:
+	// qdisc noqueue 0: root refcnt 2
+	fields := strings.Split(value, " ")
```

Comment:
We should use `strings.Fields(value)` more - it is multi-whitespace safe and more descriptive.